### PR TITLE
210 remove scheduled jobs

### PR DESCRIPTION
**Pull request by ahalberkamp**
In this pr the scheduled job was remove from the service.

Related issue: [#210](https://github.com/commercetools/commercetools-payone-integration/issues/210)

### CODE REVIEW

**Comment by heshamMassoud** [comment]
Please remove the 5min retry: https://github.com/commercetools/commercetools-payone-integration/blob/master/service/src/main/java/com/commercetools/pspadapter/payone/transaction/BaseDefaultTransactionExecutor.java#L90-L103

**Comment by ahalberkamp** [comment]
Removing the` IdempotentTransactionExecutor#executeTransaction:` is out of the scope of this pr, so a task for removing is 
 was created: https://github.com/commercetools/commercetools-payone-integration/issues/298

File: build.gradle [hunk 1]
```diff
@@ -171,9 +169,7 @@ project(":service") {
  */
 static List<String> getServiceProperties(Project project) {
     def commonProperties = [
-            'TENANTS',
-            'LONG_TIME_FRAME_SCHEDULED_JOB_CRON',
```
Comment (lojzatran) [inline build.gradle:175]: I see this variable is still being used in `.heroku.service.variables.enc`

File: functionaltests/src/test/java/specs/paymentmethods/BaseNotifiablePaymentFixture.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ public boolean receivedNotificationOfActionFor(final String paymentNames, final
 
         final String simpleClassName = getClass().getSimpleName();
         final int notificationsToWait = paymentNamesList.size();
-        LOG.info("Start waiting {} seconds for {} notifications of action [{}] for payments [{}] in [{}] test:",
+            LOG.info("Start waiting {} seconds for {} notifications of action [{}] for payments [{}] in [{}] test:",
```
Comment (lojzatran) [inline functionaltests/src/test/java/specs/paymentmethods/BaseNotifiablePaymentFixture.java:47]: Why indentation?

Comment (ahalberkamp) [reply functionaltests/src/test/java/specs/paymentmethods/BaseNotifiablePaymentFixture.java]: https://github.com/commercetools/commercetools-payone-integration/pull/226/commits/1582daf9ccec5426669fdf3ac238a7cfc74f1d23

File: build.gradle (hunk 1 above)
Comment (lojzatran) [reply build.gradle:176]: https://github.com/commercetools/commercetools-payone-integration/pull/226/commits/d6b5a41599d3e91fed30ce0b7d0e1daf00f8aea2

**Review by lojzatran — APPROVED** [review]

**Pull request merged**